edison/src/main/java/edu/illinois/cs/cogcomp/edison/features/lrec/WordConjunctionOneTwoThreeGramWindowTwo.java

```diff
@@ -66,10 +63,10 @@ public WordConjunctionOneTwoThreeGramWindowTwo(String viewName) {
             startwin = 0;
         }
 
+        int index = 0;
         for (int i = startwin; i < endwin; i++) {
-
-            window[i] = TOKENS.getConstituentsCoveringSpan(i, i + 1).get(0).getSurfaceForm();
-
+            window[index] = TOKENS.getConstituentsCoveringSpan(i, i + 1).get(0).getSurfaceForm();
```

Comment:
We assume that the span of the current constituent is 1 (hence the 2k+1). If this was a general feature extractor I would change it to `2k+(constEnd-constStart)`, but this is fine.